install.py

```diff
@@ -338,6 +329,25 @@ def install_libprologin():
     install_cfg_profile('udbsync-sub', group='udbsync_public')
 
 
+def install_sadm_secret():
+    if not os.path.exists(SECRET_PATH):
+
+        print('We need to set the Prologin SADM master secret.\n'
+              'This secret has to be shared across all the machines.')
+        # The master secret can be configured from the environment or from user input
+        secret = os.environ.get('PROLOGIN_SADM_MASTER_SECRET', None)
+        if secret is None:
+            secret = getpass.getpass('Enter ProloginSADM master secret: ')
+            secret_check = getpass.getpass('Enter ProloginSADM master secret again: ')
+            if secret != secret_check:
```

Comment:
This code hasn't been really modified, just moved to a different place. Therefore, these comments are unrelated to the change in this PR.